packages/dashboard/nginx/default.conf.template

```diff
@@ -2,6 +2,6 @@ server {
 	listen ${PORT} default_server;
 	root /usr/share/nginx/html;
 	try_files $uri /index.html;
-	sub_filter '<?- SORRY_CYPRESS_ENVIRONMENT ?>' '{ GRAPHQL_SCHEMA_URL: "${GRAPHQL_SCHEMA_URL}", CI_URL: "${CI_URL}" }';
+	sub_filter '<?- SORRY_CYPRESS_ENVIRONMENT ?>' '{ GRAPHQL_CLIENT_ATTRIBUTES: "${GRAPHQL_CLIENT_ATTRIBUTES}", GRAPHQL_SCHEMA_URL: "${GRAPHQL_SCHEMA_URL}", CI_URL: "${CI_URL}" }';
```

Comment:
Now there's a different problem - if it isn't defined, the script fails :) Try to actually run the docker image

Reply:
I was trying to find a way to have 'default' values for ngingx template. 
The problem is that if variable isn't defined, it fails. That mean we have to defined env vars in all docker-compose and helm charts. Very inconvenient. Any ideas?

Reply:
Yep :/
Any suggestions on how to proceed?

Reply:
May be just setting the defaults in the Dockerfile?

```
ENV PORT 80
ENV CI_URL https://host.docker.internal:5000
```

Do you know if docker-compose is able to override those after the image had been built?

Reply:
Yes. It will. 

So will docker run and k8s

Reply:
@fsmaia would you mind defining the defaults then.
How soon do you want this merged? I wanted to actually test it tonight

Reply:
Tomorrow would be fine to merge, I had some meetings but I'm free until the end of the day to work/help on this. Count on me :)

Reply:
Now the defaults are in the Dockerfile, should we remove them from the docker-compose files when the value equals the default?

Reply:
Sorry for delayed response. Since you've already added the variables, let's keep those!